include/aspect/mesh_deformation/fastscape.h

```diff
@@ -0,0 +1,725 @@
+/*
+  Copyright (C) 2022 by the authors of the ASPECT code.
+  This file is part of ASPECT.
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _aspect_mesh_deformation_fastscape_h
+#define _aspect_mesh_deformation_fastscape_h
+
+#include <aspect/global.h>
+
+#ifdef ASPECT_WITH_FASTSCAPE
+
+#include <aspect/mesh_deformation/interface.h>
+
+namespace aspect
+{
+  using namespace dealii;
+
+  namespace MeshDeformation
+  {
+    /**
+     * Define FastScape functions as C functions. Must use exact same function/variable name
+     * and type as used in FastScape. All function names must be made lowercase, and an
+     * underscore added at the end. Types must be defined as pointers, and sent to
+     * FastScape as a reference. Additional functions are available within FastScape,
+     * see https://fastscape.org/fastscapelib-fortran/ for a list of all functions and
+     * their input parameters.
+     */
+#ifdef __cplusplus
+    extern"C" {
+#endif
```

Comment:
This isn't strictly wrong, but I don't think you actually need these function declarations in the header file, do you? If you only need these functions in the `.cc` file, then this whole block of declarations might as well move to the `.cc` file.